neuralmonkey/decoders/transformer.py

```diff
@@ -26,13 +25,15 @@
 from neuralmonkey.tf_utils import layer_norm
 
 # pylint: disable=invalid-name
+# TODO: handle attention histories
 TransformerHistories = extend_namedtuple(
-    "RNNHistories",
+    "TransformerHistories",
     DecoderHistories,
-    [("decoded_symbols", tf.TensorArray),
-     ("self_attention_histories", List[Tuple]),
-     ("inter_attention_histories", List[Tuple]),
-     ("input_mask", tf.TensorArray)])
+    [("decoded_symbols", tf.Tensor),
+     # TODO(all) handle these!
```

Comment:
Co s tímhle, bude se to řešit v tomhle PR?

Reply:
nebude

Reply:
to bych vyřešil vzápětí, ale neni to essential aby fungoval beam search a má to potenciál to zdržet